DeepLinkKit/Router/DPLDeepLinkRouter.h

```diff
@@ -67,7 +67,10 @@ typedef void(^DPLRouteCompletionBlock)(BOOL handled, NSError *error);
  */
 - (void)registerBlock:(DPLRouteHandlerBlock)routeHandlerBlock forRoute:(NSString *)route;
 
-
+/**
+ Returns a Deep link for the given URL, if one exists. Returns `nil` if no matching routes are found for the URL.
```

Comment:
Great addition! Thanks! 

Lets stick to a similar public api documentation style such as:

```
/**
Evaluates the given URL to determine if it matches a registered route.
@param url The URL to be evaluated against registered routes.
@return A BTNDeepLink representing the URL if it matches a registered route.
*/
```